<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en"><head><meta http-equiv="Content-Type" content="text/html;charset=UTF-8"/><link rel="stylesheet" href="../jacoco-resources/report.css" type="text/css"/><link rel="shortcut icon" href="../jacoco-resources/report.gif" type="image/gif"/><title>RandoopRegressionTest0.java</title><link rel="stylesheet" href="../jacoco-resources/prettify.css" type="text/css"/><script type="text/javascript" src="../jacoco-resources/prettify.js"></script></head><body onload="window['PR_TAB_WIDTH']=4;prettyPrint()"><div class="breadcrumb" id="breadcrumb"><span class="info"><a href="../jacoco-sessions.html" class="el_session">Sessions</a></span><a href="../index.html" class="el_report">Patched-Version</a> &gt; <a href="index.source.html" class="el_package">default</a> &gt; <span class="el_source">RandoopRegressionTest0.java</span></div><h1>RandoopRegressionTest0.java</h1><pre class="source lang-java linenums">import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
<span class="fc" id="L6">public class RandoopRegressionTest0 {</span>

<span class="fc" id="L8">    public static boolean debug = false;</span>

    @Test
    public void test1() throws Throwable {
<span class="pc bpc" id="L12" title="1 of 2 branches missed.">        if (debug)</span>
<span class="nc" id="L13">            System.out.format(&quot;%n%s%n&quot;, &quot;RandoopRegressionTest0.test1&quot;);</span>
<span class="fc" id="L14">        java.util.ArrayList arrayList0 = null;</span>
        // The following exception was thrown during execution in test generation
        try {
<span class="nc" id="L17">            java.util.ArrayList&lt;java.util.ArrayList&gt; arrayListList1 = correct_java_programs.POWERSET.powerset(arrayList0);</span>
<span class="nc" id="L18">            org.junit.Assert.fail(&quot;Expected exception of type java.lang.NullPointerException; message: null&quot;);</span>
<span class="fc" id="L19">        } catch (java.lang.NullPointerException e) {</span>
            // Expected exception.
<span class="nc" id="L21">        }</span>
<span class="fc" id="L22">    }</span>

    @Test
    public void test2() throws Throwable {
<span class="pc bpc" id="L26" title="1 of 2 branches missed.">        if (debug)</span>
<span class="nc" id="L27">            System.out.format(&quot;%n%s%n&quot;, &quot;RandoopRegressionTest0.test2&quot;);</span>
<span class="fc" id="L28">        java.lang.Object obj0 = new java.lang.Object();</span>
<span class="fc" id="L29">        java.lang.Class&lt;?&gt; wildcardClass1 = obj0.getClass();</span>
<span class="fc" id="L30">        org.junit.Assert.assertNotNull(wildcardClass1);</span>
<span class="fc" id="L31">    }</span>

    @Test
    public void test3() throws Throwable {
<span class="pc bpc" id="L35" title="1 of 2 branches missed.">        if (debug)</span>
<span class="nc" id="L36">            System.out.format(&quot;%n%s%n&quot;, &quot;RandoopRegressionTest0.test3&quot;);</span>
<span class="fc" id="L37">        correct_java_programs.POWERSET pOWERSET0 = new correct_java_programs.POWERSET();</span>
<span class="fc" id="L38">        java.lang.Class&lt;?&gt; wildcardClass1 = pOWERSET0.getClass();</span>
<span class="fc" id="L39">        org.junit.Assert.assertNotNull(wildcardClass1);</span>
<span class="fc" id="L40">    }</span>
}

</pre><div class="footer"><span class="right">Created with <a href="http://www.jacoco.org/jacoco">JaCoCo</a> 0.8.7.202105040129</span></div></body></html>